scripts/collection_from_items.py

```diff
@@ -31,6 +31,12 @@ def parse_args(args: List[str] | None) -> Namespace:
     parser = argparse.ArgumentParser()
     parser.add_argument("--uri", dest="uri", help="s3 path to items and collection.json write location", required=True)
     parser.add_argument("--collection-id", dest="collection_id", help="Collection ID", required=True)
+    parser.add_argument(
+        "--published-path",
+        dest="published_path",
+        help="The path of the published dataset. Example: 's3://nz-imagery/wellington/porirua_2024_0.1m/rgb/2193/'",
```

Comment:
In other places, this was called `odr-url`. Could be good to keep it consistent.